latex2html.pl

```diff
@@ -3224,6 +3225,16 @@
 :- initialization(main, main).
 
 main(Argv) :-
+    catch_with_backtrace(main0(Argv),
+                         Error,
+                         (   set_prolog_flag(backtrace_goal_depth, 20),
```

Comment:
These 2 `set_prolog_flag` calls didn't have the desired effect ... but I couldn't find another flag for setting the option for the write.

This is what I got when running from cmake:
```
ERROR: [2,630] tex:item_commands([_484|_486],[],[_490|_492]) at /home/peter/src/swipl-devel/build.debug/home/library/ext/ltx2htm/latex2html.pl:983
ERROR: [1,904] tex:items([\(classitem,...),'\n'|...],[[...|...]|_554]) at /home/peter/src/swipl-devel/build.debug/home/library/ext/ltx2htm/latex2html.pl:961
ERROR: [1,890] tex:env(description([],['\n'|...]),[html('<dl class="latex">'),_624|...]) at /home/peter/src/swipl-devel/build.debug/home/library/ext/ltx2htm/latex2html.pl:935
```
but when I ran it by hand (calling `main0/1`), and did the debugger `u` command, I got more useful information:
```
   Exception: (1,887) tex:translate_env(description([], ['\n', \(classitem, [{'PlTerm'}]), '\n', 'Generic', 'Prolog', term|...]), file, file, _353630) ? 
```
which helped me figure out where the bad markup was.